geomesa-fs/geomesa-fs-spark-runtime/pom.xml

```diff
@@ -45,6 +45,11 @@
         </dependency>
 
         <!-- test dependencies -->
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>14.0.1</version>
+        </dependency>
```

Comment:
do you want this to be 'test' scope? i think it's compile, if you don't specify.